<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>642</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>642</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>642</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>620</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>620</height>
    </size>
   </property>
   <widget class="login" name="loginWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>620</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>620</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>620</height>
     </size>
    </property>
   </widget>
   <widget class="waitingroom" name="waitingroomWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>620</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>620</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>620</height>
     </size>
    </property>
   </widget>
   <widget class="gamewindow" name="gamewindowWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>620</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>620</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>620</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>login</class>
   <extends>QWidget</extends>
   <header>login.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>waitingroom</class>
   <extends>QWidget</extends>
   <header>waitingroom.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>gamewindow</class>
   <extends>QWidget</extends>
   <header>gamewindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
